api/modules/os/path.py

```diff
@@ -6,6 +6,17 @@ def abspath(path: str) -> str:
 """
   pass
 
+def relpath(basepath: str, targpath: str) -> str:
+  """Return the path of `targpath` relative to `basepath`. On success, the returned path will
+  always be relative to `basepath`, even if `basepath` and `targpath` share no elements. An
+  error is returned if `targpath` can't be made relative to `basepath`.
+
+  Args:
+    basepath: Path to which ``targpath`` will be made relative
+    targpath: A filesystem path
```

Comment:
Maybe "The path to be made relative to ``basepath``"? "The path to be relativized"? FWIW, https://www.geeksforgeeks.org/python-os-path-relpath-method/ pretty much just lists them both as "a path" and then lets the rest of the docs explain their meaning, rather than trying to explain each in isolation

and/or just say "whatever these words are going to be awkward regardless" and add some examples worth a thousand words, which I think would end up being much clearer and faster to read.